vllm/v1/core/sched/scheduler.py

```diff
@@ -686,23 +691,28 @@ def _make_cached_request_data(
                 token_ids = req.all_token_ids[req.num_computed_tokens:req.
                                               num_computed_tokens + num_tokens]
                 new_token_ids.append(token_ids)
+                resumed_req_token_ids.append(None)
             elif use_connector:
                 # When using a KVConnector, we add a placeholder to avoid index
                 # out of bounds errors. TODO: Remove this once the KVConnector
                 # is updated to handle token IDs properly.
                 new_token_ids.append([])
+                if resumed_from_preemption[idx]:
+                    resumed_token_ids = req.all_token_ids[:req.
+                                                          num_computed_tokens +
+                                                          num_tokens]
+                    resumed_req_token_ids.append(resumed_token_ids)
+                else:
+                    resumed_req_token_ids.append(None)
```

Comment:
maybe slightly cleaner?
```suggestion
                resumed_token_ids = None
                if resumed_from_preemption[idx]:
                    resumed_token_ids = req.all_token_ids[:req.
                                                          num_computed_tokens +
                                                          num_tokens]
                resumed_req_token_ids.append(resumed_token_ids)
```